x-pack/plugins/observability_solution/observability_onboarding/public/application/onboarding_flow_form/onboarding_flow_form.tsx

```diff
@@ -207,10 +212,19 @@ interface TitleWithIconProps {
 const TitleWithIcon: FunctionComponent<TitleWithIconProps> = ({ title, iconType }) => (
   <EuiFlexGroup responsive={false} gutterSize="m" alignItems="center">
     <EuiFlexItem grow={false}>
-      <EuiAvatar size="l" name={title} iconType={iconType} color="subdued" />
+      <EuiAvatar
+        size="l"
+        name={title}
+        iconType={iconType}
+        iconSize="l"
+        color="subdued"
+        css={{
+          padding: '24px 22px 24px 26px',
```

Comment:
Same here, can we use EUI variables for this?

Reply:
This one is to visually center the icon within the circle. The icon is not symmetrical so it looks a bit off-center without an additional adjustment. I don't think EUI constants would fit here because it's just an arbitrary 4px adjustment, I'd need to use `calc` to substract/add to one of the standard sizes, it don't think it's worth it in this case. But I'll add a comment to explain what is going on.

![CleanShot 2024-04-24 at 09 48 40@2x](https://github.com/elastic/kibana/assets/793851/1720eaac-dc79-4ea3-b16c-3813d2f1de76)

Reply:
Urgh, that's annoying - if there is no better way a comment sounds great!